assets/src/dashboard/app/views/storyAnimTool/timeline/components.js

```diff
@@ -36,7 +36,7 @@ export const AnimationList = styled.div(
     width: 100%;
     height: 100%;
     overflow: scroll;
-    background-color: ${theme.DEPRECATED_THEME.colors.gray25};
+    background-color: ${theme.colors.gray[5]};
```

Comment:
Should this be `bg.secondary`? 
<img src="https://user-images.githubusercontent.com/22185279/113633958-1e3ce700-962b-11eb-9503-ae155097c2ab.png" width="200px" />

Reply:
this component is being removed :) same with the other ones under `storyAnimTool`